### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by bizob2828**
## Proposed Release Notes
 * Updated hostname for Event API communication. (Thanks for the contribution @tmancill)
 * Added 5 retry polling for log event server test to reduce failure rate. It seems it can occasionally take > 5 seconds     for the log events to be retrieved.
 * Added Node 16 to CI.
 * Resolved npm audit warning.
 * Added husky and a pre-commit script to automatically update the 3rd party manifest and notices when new packages are i    nstalled/updated.

## Links
https://github.com/newrelic/newrelic-telemetry-sdk-node/pull/63
https://github.com/newrelic/newrelic-telemetry-sdk-node/pull/64
https://github.com/newrelic/newrelic-telemetry-sdk-node/pull/65

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+* Added 5 retry polling for log event server test to reduce failure rate. It seems it can occasionally take > 5 seconds for the log events to be retrieved.
+* Added Node 16 to CI.
+* Resolved npm audit warning.
+* Added husky and a pre-commit script to automatically update the 3rd party manifest and notices when new packages are installed/updated.
+
 ## 0.4.2 (2021-05-17)
 * Added batchSize interface to LogBatch class.
 
 ## 0.4.1 (2021-05-12)
-* The logs client client options, batch and log models can be referenced with telemetry.logs.*.
```
Comment (michaelgoin) [inline CHANGELOG.md:7]: Errant stripping of a wildcard indicator here.

Comment (michaelgoin) [reply CHANGELOG.md:7]: Or if intentionally don't want, need to remove a period. But seems like the intent is more straightforward with the indicator.

Comment (bizob2828) [reply CHANGELOG.md:7]: ah crap.  it was messing up the markdown i now see what that was indicating. ill wrap in ``

**Review by michaelgoin — APPROVED** [review]
👍

**Pull request merged**